packages/passport/sdk/src/zkEvm/user/loginZkEvmUser.ts

```diff
@@ -25,13 +25,8 @@ export const loginZkEvmUser = async ({
   multiRollupApiClients,
   jsonRpcProvider,
 }: LoginZkEvmUserInput): Promise<LoginZkEvmUserOutput> => {
-  let user = null;
-  try {
-    user = await authManager.getUser();
-  } catch (err) {
-    // eslint-disable-next-line no-console
-    console.warn('eth_requestAccounts` failed to retrieve a cached user session:', err);
-  }
+  let user = await authManager.getUser();
```

Comment:
Why remove the try-catch?

Reply:
Was leftover from previous approach before I implemented feedback. Will restore it